newsroom/commands/__init__.py

```diff
@@ -1,24 +1,27 @@
-from .create_user import create_user  # noqa
-from .elastic_rebuild import elastic_rebuild  # noqa
-from .elastic_init import elastic_init  # noqa
-from .content_reset import content_reset  # noqa
-from .index_from_mongo import index_from_mongo, index_from_mongo_period  # noqa
-from .remove_expired import remove_expired  # noqa
-from .alerts import (  # noqa
+# disable "imported but unused" complains
+# flake8: noqa: F401
```

Comment:
From what I read flake8 doesn't support, and don't want to support, file level rule exclusions.
Does this work, or does this disable flake8 for this entire file for all rules?

Reply:
You're right. I just double checked it and indeed it disabled flake8 entirely for the whole file. Instead, what I did was to add a `per-file-ignores` in the `setup.cfg` because it really bothered me to see the #noqa so many times.